src/test/java/org/fundacionjala/coding/nestor/CustomerTest.java

```diff
@@ -0,0 +1,36 @@
+package org.fundacionjala.coding.nestor;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class CustomerTest {
+    private Customer customer;
+
+    @Test
+    public void testCustomerName() {
+        customer = new Customer("Customer Name");
+        assertEquals(customer.getName(), "Customer Name");
+    }
+
+    @Test
+    public void testAmountForOneMovieRental() {
+        customer = new Customer("Customer Name");
+        Rental terminator = new Rental(new Movie("Terminator", 0), 2);
+        customer.addRental(terminator);
+        //assertEquals(customer.getAmountFor(terminator), 2.0, 0.0);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)